main.cpp

```diff
@@ -108,7 +108,410 @@ write 3 UDTs below that EACH have:
  */
 
 #include <iostream>
+#include <vector>
+#include <string>
+#include <unordered_set>
+
+struct Loan
+{
+    double loanAmount;
+    double interestRateInPercent;
+    unsigned long numPayments;
+    bool isSecured;
+    std::vector<double> payments;
```

Comment:
containers have not been introduced yet.  They're introduced in Project 8.
please refrain from using language features that haven't been introduced in the course yet. 
- [ ] revise